### ISSUE TITLE
wallrunning rarely causes you to instantly jump from it

### ISSUE THREAD

**Issue by shuk-fuk** [ScrewdriverHyena/ParkourFortress-Bugs#100]
it doesn't happen too often which is why it'd be hard to get footage of it but it's definitely a thing that happens. it sort of feels like that one suggestion i made to add a wallrun+jump command for fixing that one wallrun bug.

**Comment by ScrewdriverHyena** [comment]
Really unsure about this one, I'll be honest, I'm going to need at least a demo before really looking into it, which I know is tough because it happens randomly, but that would help me rule out a lot if possible.

**Comment by ScrewdriverHyena** [comment]
Dropping this here, possibly a video of the issue sent to me by ratlas (third wallrun) https://streamable.com/aacus6

**Comment by shuk-fuk** [comment]
yeah, looks like it. he tried to wallrun cancel but the bug is just so quick that he couldn't do it. i dont think it matters if you're crouched or not.

**Comment by wuhesk** [comment]
Seems to happen more consistently when these strings of events happen (possibly around or on the same ticks):
> -duck, +jump (wallrun)
> -duck, speedboost, +jump (wallrun)

Video footage (also posted at #bug-report-discussion):
[https://cdn.discordapp.com/attachments/888830493984972810/888830618157326416/bug.webm](url)

**Comment by Serafeline** [comment]
This is currently being tracked - https://dev.azure.com/NBS-Games/NoBrightShadows-General/_workitems/edit/1107
